python/utils/terra_util.py

```diff
@@ -186,6 +196,58 @@ def update_user_acl(
             )
         return request_json
 
+    def update_multiple_users_acl(self, acl_list: list[dict]) -> dict:
+        """acl list should be a list of dictionaries with the following format:
+        [{"email": "email", "accessLevel": "access_level", "canShare": "can_share", "canCompute": "can_compute"}]"""
+        url = f"{self.TERRA_LINK}/workspaces/{self.billing_project}/{self.workspace_name}/acl"
+        logging.info(
+            f"Updating users in workspace {self.billing_project}/{self.workspace_name}")
+        response = self.request_util.run_request(
+            uri=url,
+            method=PATCH,
+            content_type="application/json",
+            data=json.dumps(acl_list)
+        )
+        return response.json()
```

Comment:
Same thing here with the response code. I think it can make debugging easier, that way we know if it's the API endpoints that are failing vs. our code.